### ISSUE TITLE
Where is the corresponding file for DN-FPN?

### ISSUE THREAD

**Issue by zzhangfan** [hoiliu-0801/DNTR#3]
Thank you very much for open-sourcing the paper! I have a question: the DN-FPN related code files are not in the NECK folder. So, can you help me?

**Comment by hoiliu-0801** [comment]
Hello @zzhangfan thank you for your interest in DNTR. Please check "mmdet-dntr/mmdet/models/detectors/two_stage.py".

**Comment by zzhangfan** [comment]
Thank you very much for your reply. In the two_stage.py file, I noticed that you multiplied both geo_loss and sem_loss by 0. However, in the paper, these two losses are weighted by 0.1. Therefore, is it necessary to modify the weight to 0.1 to use DN-FPN?

**Comment by hoiliu-0801** [comment]
Yes, you are correct. I have reverted it back to 0.1. Please note that the loss tends to become NaN due to the older version of mmdetection. I suggest training with a coefficient of 0 for both losses to obtain a pre-trained model. Then, resume training with a coefficient of 0.1 to achieve the final result. I will also release the pre-trained weight in the upcoming weeks.
